addons/l10n_uy_website_sale/controllers/portal.py

```diff
@@ -0,0 +1,44 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.portal.controllers.portal import CustomerPortal
+from odoo.http import request
+
+
+class CustomerPortalUruguay(CustomerPortal):
+
+    def _is_uruguay_company(self):
+        return request.env.company.country_code == 'UY'
```

Comment:
Fiscal country?

Reply:
Not sure some localisation use direct country and only one BR uses a fiscal country. Try to know difference but not luck :(
(If we must use Fiscal country then have to probably change other E-commerce related localisations also :thinking: )